crates/miden-lib/src/transaction/memory.rs

```diff
@@ -295,7 +287,7 @@ pub const NATIVE_ACCT_PROCEDURES_CALL_TRACKING_PTR: MemoryAddress =
 
 /// The offset at which the number of storage slots contained in the account storage is stored
 /// relative to the start of the account data segment.
-pub const NUM_ACCT_STORAGE_SLOTS_OFFSET: MemoryAddress = 2340;
+pub const NUM_ACCT_STORAGE_SLOTS_OFFSET: MemoryAddress = 1316;
```

Comment:
Not from this PR, but let's rename:
- `NUM_ACCT_STORAGE_SLOTS_OFFSET` -> `ACCT_NUM_STORAGE_SLOTS_OFFSET`
- `NUM_ACCT_PROCEDURES_OFFSET` -> `ACCT_NUM_PROCEDURES_OFFSET`

To be more consistent. This also applies to the constants of the same name in `memory.masm`.